x/staking/keeper/unbonding.go

```diff
@@ -12,23 +12,14 @@ import (
 
 // IncrementUnbondingID increments and returns a unique ID for an unbonding operation
 func (k Keeper) IncrementUnbondingID(ctx context.Context) (unbondingID uint64, err error) {
-	store := k.storeService.OpenKVStore(ctx)
-	bz, err := store.Get(types.UnbondingIDKey)
+	unbondingID, err = k.UnbondingID.Next(ctx)
```

Comment:
Isn't Next getting and setting the ID directly? 
https://github.com/cosmos/cosmos-sdk/blob/dfb332da268cdebb81a4a103ab41eeff7572e9e6/collections/sequence.go#L35-L43